spec/spec_helper.rb

```diff
@@ -22,6 +22,9 @@
       conf.app_secret = 'APP_SECRET'
       conf.base_url = 'http://www.mifiel.com/api/v1'
     end
+
+    # Creates ~/tmp folder so signed files can be saved correctly
+    FileUtils.cd('.') { |dir| FileUtils.mkdir_p('tmp') }
```

Comment:
The `FileUtils.cd('.')` will stay in the current dir (the dot is CWD), but the comment says it's creating `~/tmp`, so either the comment or the implementation is incorrect.

If you did want to put it in `./tmp`, you can do:

```ruby
Dir.mkdir "tmp" unless File.directory? "tmp"
```

And if you did want to put it in ~/tmp, you can do:

```ruby
dir = File.join Dir.home, "tmp"
Dir.mkdir dir unless File.directory? dir
```

So that you don't have to add a dependency on fileutils.

Reply:
Thanks @JoshCheek yes with your suggestion we can definitely remove FileUtils dependency. 🙌🏻